lib/npm/test.js

```diff
@@ -55,8 +55,13 @@ function test(context, next) {
       npmBin = path.join(path.dirname(npmBin), 'node_modules', 'npm', 'bin',
           'npm-cli.js');
     }
-    const proc = spawn(nodeBin, [npmBin, 'test', '--loglevel=' +
-        context.options.npmLevel], options);
+    // Run `npm test`, or `/path/to/customTest.js` if the customTest option
+    // was passed.
+    const args = context.options.customTest ?
+      context.options.customTest.split(' ') :
```

Comment:
is this assuming that customTest will always be a string? Would we want to potentially pass a path to a file?

Reply:
Yeah, we shouldn't be splitting here, see https://github.com/nodejs/citgm/pull/553#discussion_r172970245.

@al-k21 added the split back in to debug a test issue he was seeing, should be fixed now.

Reply:
Yes, `args` should be an array with one element (the path to the customTest.js script). 
This now fixed.